doc/sphinx-guides/source/installation/config.rst

```diff
@@ -4462,16 +4462,22 @@ A comma separated list of Category/Tag names defining the order in which files w
 The setting can include custom tag names along with the pre-defined tags(Documentation, Data, and Code are the defaults but the :ref:`:FileCategories` setting can be used to use a different set of tags).
 The default is category ordering disabled.
 
+``curl -X PUT -d 'Code,Data,Documentation' http://localhost:8080/api/admin/settings/:CategoryOrder``
```

Comment:
Should the order be the same as above? "Documentation,Data,Code"?

I'm fine either way.